### ISSUE TITLE
Add new information product types

### ISSUE THREAD

**Issue by alanorth** [AgriculturalSemantics/cg-core#42]
In the context of harmonizing metadata in preparation for closer integration between information systems in the One CGIAR, a question was raised about adding more types of information products. The [current list of types](https://agriculturalsemantics.github.io/cg-core/IPtypes.html) in CG Core is somewhat of a mashup of types historical used in various repositories, but should be amended to include more types based on current reality on the ground.

**Comment by marieALaporte** [comment]
I removed the types that were both in CGSpace and CG Core. Below are my comments
I was able to dig up some definition from what we did with the MWG when we started our assessment of existing standards. Could someone validate what is in the table and provide missing information. Tagging @ayabowork for your information
|Type|Comments|
|------|----------|
|Abstract|proposed def: A brief summary of a work on a particular subject, designed to act as the point-of-entry that will help the reader quickly to obtain an overview of the work's contents. The abstract may be an integral part of the work itself, written by the same author(s) and appearing at the beginning of a work such as a research paper, report, review or thesis. Alternatively it may be separate from the published work itself, and written by someone other than the author(s) of the published work, for example by a member of a professional abstracting service such as CAB Abstracts. Source: FaBiO|
|Blog Post|there is post. I updated its definition.|
|Brief|proposed def: Brief description of important new research, also known as “communication”. Source: MEL|
|Case Study|proposed def: A qualitative descriptive research study of individuals or a group. Source: VIVO|
|Data Paper|A data paper is a scholarly publication describing a particular dataset or group of dataset, published in the form of a peer-reviewed article in a scholarly journal. The main purpose of a data paper is to describe data, the circumstances of their collection, and information related to data features, access and potential reuse. Source: COAR Resource Types|
|Dataset|we have data file and Dataset As A Collection Of Data|
|Equation|proposed definition: Strings of mathematical expressions and formulas (e.g., NDVI, EVI, LSWI). Source: MEL|
|Extension Material|could you provide a definition for that type?|
|Internal Document|Internal report is a record of findings collected for internal use. It is not designed to be made public and may include confidential or proprietary information. Source: MEL|
|Journal Item|we have press item|
|Logo|Specific logos related to initiatives, institutions, service providers or products. Source: MEL|
|Manuscript-unpublished|this should not be a type. Should just be an abstract or a journal article with in the Core Metadata the field peer reviewed to false (or missing). One can argue that if it is on cgspace, that's not unpublished.... There is also a working paper type that might work for what is needed|
|Mobile Message|Message circulated via mobile phone (e.g. SMS, MMS). Source: MEL|
|News Item|Articles in a daily, weekly or monthly publication reporting on news and social issues aimed at the public. May entail critical analysis based on expertise in the field. Source: MEL|
|Opinion Piece|could you provide a definition for that type?|
|Photo Report|how is it different from Image?|
|Report|we have financial, technical and research report. Would it be possible to reassign this generic type to the more specific types?|
|Template|Electronic files with a predesigned, customized format and structure, as for a fax, letter, or expense report, ready to be filled in. Also, questionnaires.|
|Training Material|could you provide a definition for that type?|
|Wiki|not sure how that's different from website|
|Other|that's more for you at a system to have such option. It cannot really be in a standard|

**Comment by monday2016** [comment]
Here are the types used by MEL 
[MEL-Types-2022-06-24-13-04-37_MS.xlsx](https://github.com/AgriculturalSemantics/cg-core/files/9029682/MEL-Types-2022-06-24-13-04-37_MS.xlsx)
 
@valentina-dc FYI

**Comment by marieALaporte** [comment]
@monday2016 do you need to have things classified as types and subtypes, or one list with all the types enough?

**Comment by alanorth** [comment]
Thank you for the analysis, @marieALaporte!

A few comments on @monday2016's MEL list:

- `Blog` presumably refers to a Blog Post, not an entire Blog (which would be a website). Recommend using "Blog Post" here.
- `Donor Report` should be a sub-type of "Report"
- There are few other types that might also be sub-types of "Report", for example: `Concept Note` and `Proposal`. These are not sufficiently distinct types of written work that warrant a new type in my opinion (whereas `Brief` and `Brochure` are widely known outside of academia and even usually have a distinct physical format)
- `News Item/Press Item` we have these on CGSpace, but they are separated. I think we should determine if these are the same thing and merge them with one name, or separate them if they are truly distinct
- `Tool` is confusing because in my experience it usually refers to a document describing some method for doing something, rather than a piece of Software or a literal tool like a hammer ;)
- I agree `Other` doesn't really belong in a standard lookup list, but systems can use it if there is no matching option

I can see how the sub types are useful and informative, but I don't know how we incorporate them.

**Comment by monday2016** [comment]
@alanorth Please see my replies here below. Hope all is clear now.

> * `Blog` presumably refers to a Blog Post, not an entire Blog (which would be a website). Recommend using "Blog Post" here. You are right but we used the type "Blog" as we do not have "Blog Post"
> * `Donor Report` should be a sub-type of "Report". I agree with you, it is sub-type of the report and it is at the same time a distinct type as it is used for project reporting under one specific section "donor reports".
> * There are few other types that might also be sub-types of "Report", for example: `Concept Note` and `Proposal`. These are not sufficiently distinct types of written work that warrant a new type in my opinion (whereas `Brief` and `Brochure` are widely known outside of academia and even usually have a distinct physical format). "Concept notes" and "Proposals" are added for project planning and CRP planning purposes as we have a project management part in MEL. 
> * `News Item/Press Item` we have these on CGSpace, but they are separated. I think we should determine if these are the same thing and merge them with one name, or separate them if they are truly distinct. I agree with you.
> * `Tool` is confusing because in my experience it usually refers to a document describing some method for doing something, rather than a piece of Software or a literal tool like a hammer ;). Tools mostly are used as you mentioned in a document describing some methods.
> * I agree `Other` doesn't really belong in a standard lookup list, but systems can use it if there is no matching option

I am also copying here below some new suggested types to be added to MEL

- Guideline
- Infographic  which is in CG Core already
- Factsheet

@enricobonaiuti and @valentina-dc FYI

**Comment by MargaritaRamirez** [comment]
Hi everyone, 
We want to let you know that we were working with Marie Angelique since last year, evaluating the way to align the categories that we have in MARLO with the ones shared in the metadata working group. 
We export the information and made a first version of the mapping and last week she shared and contextualize with me what you have been discussing. 
Considering this we  export the type information from CG Core, CGSpace, and MEL and mapped again
You can find the information at this link https://cgiar.sharepoint.com/:x:/s/CCAFS-KDS/EbOlrESU5DdHlb16BZ_66lsBuvmr4O4cNCnJz5qVReLYcw?e=8ddKrw
sheet 2-Mapping V.2 colored table.

**Comment by alanorth** [comment]
@MargaritaRamirez that's a useful comparison, thank you. I have sorted the CG Core, CGSpace, and MEL columns alphabetically so it is easier to compare types.

Also, I propose we remove "Other" from all lists immediately since it doesn't belong in a public controlled vocabulary and should be an internal decision at each repository. CGSpace has 249 items with type "Other", FYI. :thinking:

**Comment by marieALaporte** [comment]
Should we organize a meeting to discuss the next steps? Would be great to have something online for the types sooner rather than later.

**Comment by alanorth** [comment]
@marieALaporte yes let's meet to discuss. I've been on leave for a bit. The earliest I could be available would be the week of Monday, August 29th. We could move this to email for the scheduling?

**Comment by valentina-dc** [comment]
Dear @alanorth, @marieALaporte, and @MargaritaRamirez,
Thank you very much for your discussion and contribution to this issue.

We have run a similar exercise of matching the knowledge products types (it looks like we have all done a comparable exercise!) that has helped us to carry on a discussion within our MEL Team.

Please find [at this link the shared file](https://docs.google.com/spreadsheets/d/1HpjUJVPl2lB4GQTM3k9N_WCXLZKfnarwyovh1b09NTs/edit?usp=sharing). Results of our discussion are in column D. We have also reflected with Marie Angelique on some of these types and on the need for harmonization and definitions.

Some general comments for the discussion we will have in two weeks:
1. Aim to simplify the list as much as possible and reflect if some types could become sub-types and if this could be feasible. For example, a Data Paper could be a sub-type of a Journal Article or a Technical Report, a sub-type of Report.
2. See if there is an actual use for all these types. How many use cases do we have for a specific type? This might guide the decision on removing a type or adding sub-types, ...
3. We should revise some of the definitions together. E.g., what is the difference between News items and Press Items? 
4. Lastly, are there any existing standards we could use as a reference? @marieALaporte and @MargaritaRamirez have mentioned that the Metadata WG reviewed current standards. @marieALaporte, could you please send us the link to this review? Is there any aspects or recommendation we should consider?

In case you have any comments on the Excel, please feel free to add a column.

I am looking forward to our upcoming meeting.

FYI @monday2016, @enricobonaiuti, @svetlanasaakova, @ayabowork

**Comment by ayabowork** [comment]
Hi all, here is link to the document that the MWG was working on to harmonize output types, authors, journal titles etc across CGSpace, CG Core and CGIAR repositories: https://cgiar.sharepoint.com/:x:/r/sites/metadata-wg/_layouts/15/Doc.aspx?sourcedoc=%7B4C74F24A-D100-4D48-A034-2A9658C27528%7D&file=2021_harmonizing_controlled%20lists.xlsx&action=default&mobileredirect=true&cid=86e0bdab-5211-465b-8566-f853c4f35295  It looks like there is some duplication of effort across centers so would be nice to work on this together. @valentina-dc , @alanorth, @MargaritaRamirez, @marieALaporte
